src/sentry/static/sentry/app/components/selectMembers/index.tsx

```diff
@@ -77,7 +77,7 @@ type FilterOption<T> = {
  * A component that allows you to select either members and/or teams
  */
 class SelectMembers extends React.Component<Props, State> {
-  static propTypes = {
+  static propTypes: any = {
```

Comment:
ts 4 picks up the prop types for some reason and starts trying to type check with them. Use props instead?

Reply:
You could see if any jsx files still use it and maybe just remove it completely

Reply:
removed